### PR TITLE
KEA-210 As a user, I want to see criteria for each Group so that I know which group to join

### PR DESCRIPTION
**Pull request by jiahn326**
KEA-210 As a user, I want to see criteria for each Group so that I know which group to join

### CODE REVIEW

File: backend/config/sync/core-store.plugin_content_manager_configuration_content_types##api##group.group.json [hunk 1]
```diff
@@ -76,6 +76,20 @@
           "sortable": true
         }
       },
+      "Criteria": {
+        "edit": {
+          "label": "Criteria",
+          "description": "",
+          "placeholder": "",
```
Comment (doolri1276) [inline backend/config/sync/core-store.plugin_content_manager_configuration_content_types##api##group.group.json:83]: Can we have some examples as placeholders? You can get examples from the Figma :)

File: backend/config/sync/admin-role.admin-lrh30qnl.json [hunk 2]
```diff
@@ -313,7 +313,8 @@
         "fields": [
           "Name",
           "ShortDescription",
-          "Picture"
```
Comment (doolri1276) [inline backend/config/sync/admin-role.admin-lrh30qnl.json:316]: No permissions for Criteria

File: backend/config/sync/admin-role.strapi-author.json [hunk 3]
```diff
@@ -260,7 +260,8 @@
         "fields": [
           "Name",
           "ShortDescription",
-          "Picture"
```
Comment (doolri1276) [inline backend/config/sync/admin-role.strapi-author.json:263]: No permissions for Criteria

File: backend/src/api/group/content-types/group/schema.json [hunk 4]
```diff
@@ -59,7 +59,16 @@
         "Community",
         "Education",
         "Activities"
```
Comment (doolri1276) [inline backend/src/api/group/content-types/group/schema.json:61]: Can there be one more enum "Music"? before Activities

File: backend/src/api/group/content-types/group/schema.json [hunk 5]
```diff
@@ -31,7 +31,7 @@
           "localized": true
         }
       },
-      "type": "string",
+      "type": "text",
```
Comment (doolri1276) [inline backend/src/api/group/content-types/group/schema.json:34]: In my opinion, this should stay as string

File: frontend/src/components/GroupCard/GroupCard.tsx [hunk 6]
```diff
@@ -47,6 +47,9 @@ export const GroupCard: React.FC<GroupCardProps> = (props) => {
           <Skeleton colorMode="light">
             <H4 style={h4}>{props.Name}</H4>
           </Skeleton>
+          <Skeleton colorMode="light">
+            <P style={bodyNormal}>{props.Criteria}</P>
```
Comment (doolri1276) [inline frontend/src/components/GroupCard/GroupCard.tsx:51]: The style is not bodyNormal.

**Review by doolri1276 — CHANGES_REQUESTED** [review]

File: backend/config/sync/admin-role.admin-lrh30qnl.json
Comment (doolri1276) [inline backend/config/sync/admin-role.admin-lrh30qnl.json:1]: Please rebase

File: backend/src/api/group/content-types/group/schema.json (hunk 5 above)
Comment (jiahn326) [reply backend/src/api/group/content-types/group/schema.json:34]: Fixed it, thank you!

File: backend/src/api/group/content-types/group/schema.json (hunk 4 above)
Comment (jiahn326) [reply backend/src/api/group/content-types/group/schema.json:62]: I added it👍

File: backend/config/sync/admin-role.strapi-author.json (hunk 3 above)
Comment (jiahn326) [reply backend/config/sync/admin-role.strapi-author.json:263]: I added it. thank you!

File: backend/config/sync/admin-role.admin-lrh30qnl.json (hunk 2 above)
Comment (jiahn326) [reply backend/config/sync/admin-role.admin-lrh30qnl.json:316]: Added it. Thank you!

File: frontend/src/components/GroupCard/GroupCard.tsx (hunk 6 above)
Comment (jiahn326) [reply frontend/src/components/GroupCard/GroupCard.tsx:51]: I don't know the information about the style. Currently, I changed to bodyLarge. Please let me know if it it is not correct.

File: backend/config/sync/admin-role.admin-lrh30qnl.json
Comment (jiahn326) [reply backend/config/sync/admin-role.admin-lrh30qnl.json:1]: Please let me know if I did rebase properly. Thank you!

File: backend/config/sync/core-store.plugin_content_manager_configuration_content_types##api##group.group.json (hunk 1 above)
Comment (doolri1276) [reply backend/config/sync/core-store.plugin_content_manager_configuration_content_types##api##group.group.json:83]: placeholder is not a description. Placeholder is to give an example you can put text like this.

ex) For wives, For members age 19+

File: frontend/src/components/GroupCard/GroupCard.tsx (hunk 6 above)
Comment (doolri1276) [reply frontend/src/components/GroupCard/GroupCard.tsx]: We can go through this during today's meeting. :)

File: backend/config/sync/admin-role.admin-lrh30qnl.json
Comment (doolri1276) [reply backend/config/sync/admin-role.admin-lrh30qnl.json:1]: Your rebase was correct. Great job.

Your other PR KEA-145 is merged now, so you'll need to perform rebase again.
Whenever there is any PR merged to develop, you would need to do rebase each time :)

**Review by doolri1276 — CHANGES_REQUESTED** [review]

File: backend/config/sync/core-store.plugin_content_manager_configuration_content_types##api##group.group.json (hunk 1 above)
Comment (jiahn326) [reply backend/config/sync/core-store.plugin_content_manager_configuration_content_types##api##group.group.json:83]: Sorry for my misunderstanding and thank you for the details! I made changes.

File: backend/config/sync/admin-role.admin-lrh30qnl.json
Comment (jiahn326) [reply backend/config/sync/admin-role.admin-lrh30qnl.json:1]: okay thank you about the info :)

**Review by doolri1276 — APPROVED** [review]

**Pull request merged**